presto-main/src/main/java/com/facebook/presto/type/CharOperators.java

```diff
@@ -92,6 +96,134 @@ public static boolean between(@SqlType("char(x)") Slice value, @SqlType("char(x)
         return compareChars(min, value) <= 0 && compareChars(value, max) <= 0;
     }
 
+    private static byte toUpperCase(byte b)
+    {
+        return isLowerCase(b) ? ((byte) (b - 32)) : b;
+    }
+
+    private static boolean isLowerCase(byte b)
+    {
+        return (b >= 'a') && (b <= 'z');
+    }
+
+    @LiteralParameters("x")
+    @ScalarOperator(CAST)
+    @SqlType(StandardTypes.BOOLEAN)
+    public static boolean castToBoolean(@SqlType("char(x)") Slice value)
```

Comment:
I think the best place for character string casts would be `com.facebook.presto.operator.scalar.CharacterStringCasts`